Sources/Build/PackageVersion.swift

```diff
@@ -15,7 +15,7 @@ import class Utility.Git
 import struct Utility.Path
 
 public func generateVersionData(_ rootDir: AbsolutePath, rootPackage: Package, externalPackages: [Package]) throws {
-    let dirPath = rootDir.appending(".build/versionData")
+    let dirPath = rootDir.appending(components: ".build", "versionData")
```

Comment:
Cool! I didn't do all of them under Tests, but I also prefer this style (it also is much more readable when some of the components include string interpolation) so we can try and migrate everything toward it.